quilt/tools/package.py

```diff
@@ -191,23 +195,38 @@ def save_df(self, df, name, path, ext, target):
             elif parqlib is ParquetLib.ARROW:
                 table = pa.Table.from_pandas(df)
                 parquet.write_table(table, storepath)
+            elif parqlib is ParquetLib.SPARK:
+                assert isinstance(df, sparksql.DataFrame)
+                df.write.parquet(storepath)
             else:
                 assert False, "Unimplemented ParquetLib %s" % parqlib
         else:
             assert False, "Unimplemented PackageFormat %s" % enumformat
 
         # Move serialized DataFrame to object store
-        filehash = digest_file(storepath)
-        self._add_to_contents(buildfile, filehash, ext, path, target)
-        os.rename(storepath, self._object_path(filehash))
+        if os.path.isdir(storepath): # Pyspark
+            hashes = []
+            files = [file for file in os.listdir(storepath) if file.endswith(".parquet")]
```

Comment:
Use something other than `file`: in Python2, it will clobber the `file` built-in.